packages/project-types/src/package.ts

```diff
@@ -0,0 +1,19 @@
+export interface Package {
```

Comment:
It's odd that we define the package json type. Should we use something existing e.g. https://www.npmjs.com/package/type-fest and only extend it with UI5 / Fiori tools specific things?

Reply:
@tobiasqueck, I changed this to use type from `PackageJson` `type-fest` and extend it with our custom properties. As I have it as a `devDependency` and import it using 

```
import type { PackageJson } from 'type-fest';
```

it should not add anything it terms of size to `@sap-ux/project-types`. However, I've checked `type-fest` by installing it into an empty folder and it takes ~400KB, what concerns me slightly, as we replace 7 lines of code in our type definition. We could think of just adding a file like https://github.com/sindresorhus/type-fest/blob/main/source/package-json.d.ts which contains just the type for `package.json`, but then as I said, it doesn't really affect the resulting module size, it just uses some resources for developers.

Reply:
I am also ok with copying (but don't forget to list it in https://github.com/SAP/open-ux-tools/blob/feat/293/project-access/.reuse/dep5. 
Otherwise, I think the `type-fest` module is pretty interesting, and I am sure we find my uses for it.